docs/guides/getting-started-orchestrate-human-tasks.md

```diff
@@ -45,10 +47,12 @@ Variables are part of a process instance and represent the data of the instance.
 ### Implement a form
 
 1. Navigate to the Modeler **Home** page and click the project you are working on. Then, click **New > Form**.
-2. Name your form. In this case, we've named ours **Decide what's for dinner**.
-3. Click and drag the **Select** element onto the palette. Give this **Select** field a description within the properties panel. We've described ours as **What's for dinner?**
-4. Scroll down to the **Values** section of the properties panel to add your values. For our dinner add an extra value by clicking on the plus sign. Enter the value **Chicken** with the label as **Chicken** and enter the value **Salad** with the label as **Salad** in the other value.
+2. Name your form. In this case, we've named ours `Dinner Options`.
+3. Click and drag the **Select** element onto the palette. Give this **Select** field a description within the properties panel. We've labeled ours as `What's for dinner?`. Additionally, we need to set a key which maps to a process variable. As already defined by the conditions in the process model we are going to use the variable `meal`.
+4. Scroll down to the **Values** section of the properties panel to add your values. For our dinner add an extra value by clicking on the plus sign. Enter the value `Chicken` with the label as `Chicken` and enter the value `Salad` with the label as `Salad` in the other value.
```

Comment:
Should the value be upper or lower case?

Reply:
In my humble opinion the value (e.g. Salad) should be lower case (since it is a value of the variable) and the label upper case (since it is shown in the UI) 

Since this is all over the place like this I decided to leave it for now since it is just a "cosmetic" issue which does not lead to any inconsistency. 

I am happy to fix it though. Lmk what you think about it. :) 
Thanks for the review!

Reply:
Could you sync with @christinaausley on this and handle it in a separate PR? It would be good for her to have the context and maybe audit the rest of our docs for consistency.